docs/setup.rst

```diff
@@ -468,8 +468,37 @@ writing. Windows is not and will never be supported.
     to compile this by yourself, because this software has been patented until around 2017 and
     binary packages are not available for most distributions.
 
-Migration to paperless-ngx
-#########################
+Migrating to Paperless-ngx
+##########################
+
+Migration is possible from both Paperless-ng or directly from the 'original' Paperless.
+
+Migrating from Paperless-ng
+===========================
+
+Paperless-ngx is meant to be a drop-in replacement for Paperless-ng and thus upgrading should be
+trivial for most users, especially when using docker. However, as with any major change, it is 
+recommended to take a full backup first. Once you are ready, simply change the docker image to
+point to the new source, e.g. change:
```

Comment:
I think this could be more explicit. Can we show a patch? Also we should say which file to change (`docker-compose.yml`).